test/tasks/karpenter-test-suite.yaml

```diff
@@ -0,0 +1,127 @@
+# ginkgo-suite is a generic task that executes ginkgo on the given repository and ginkgo suite.
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: karpenter-test-suite
+  namespace: karpenter-tests
+spec:
+  description: |
+    Execute a karpenter test suite
+  params:
+    - name: cluster-name
+      default: $(context.taskRun.name)
```

Comment:
Can we pre-fix this with more metadata? If we have many of these running in our cluster, we'll need an easier way to distinguish different which cluster was created with which parameters.

Reply:
It should have pipeline-name-task-name prefixed once this is fully wired up.

Reply:
Tekton allows UID, but the strings get so long that IAM starts to complain (max len 64)